frontend/src/pages/projects/screens/spawner/imageSelector/ImageStreamSelector.tsx

```diff
@@ -91,10 +92,9 @@ const ImageStreamSelector: React.FC<ImageStreamSelectorProps> = ({
                 }
                 onClick={() => onImageStreamSelect(imageStream)}
                 icon={
-                  <img
-                    style={{ height: 25 }}
-                    src={typedObjectImage(ProjectObjectType.project)}
-                    alt=""
+                  <TypedObjectIcon
+                    resourceType={ProjectObjectType.project}
+                    style={{ height: 24, width: 24 }}
```

Comment:
This has been addressed in this PR: https://github.com/opendatahub-io/odh-dashboard/pull/4149/files#diff-99df0b9ddc80bb2a9adaeb10509a08e6049b73ea8632a0e2f480880847d9e12fR94